src/engine/strat_engine/filesystem.rs

```diff
@@ -109,6 +169,30 @@ impl Recordable<FilesystemSave> for StratFilesystem {
     }
 }
 
+/// Return total bytes, total bytes used
+pub fn fs_usage(mount_point: &PathBuf) -> EngineResult<(Bytes, Bytes, Bytes)> {
+    let mut stat = Statvfs::default();
+    try!(statvfs(mount_point, &mut stat));
+    let total_bytes = stat.f_blocks * stat.f_bsize;
+    Ok((Bytes(total_bytes), Bytes(total_bytes - stat.f_bfree * stat.f_bsize), Bytes(stat.f_bsize)))
```

Comment:
Seems like it would make more sense to return the free bytes and let the caller decide how it wants to use that information.